src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -137,6 +137,9 @@ def __read_with_appropriate_encoding(file_path, format_):
         with io.open(file_path, 'r', encoding=default_encoding) as config_file:
             if format_ == 'json':
                 config_data = json.load(config_file)
+                # Only accept json objects
+                if not isinstance(config_data, dict):
+                    raise ValueError("Json object required but type '{}' was given.".format(type(config_data).__name__))
```

Comment:
Should we add this validation for yaml files too?

Reply:
Currently, YAML file data that is not formatted as an object is rejected already. An attempt to import a YAML file with array data throws the error shown below.
![image](https://user-images.githubusercontent.com/102393878/183215402-703e73ac-8ece-415a-bc89-fd04f7e97956.png)